src/index.js

```diff
@@ -1,5 +1,7 @@
 const toMatchStyledComponentsSnapshot = require('./matchers/toMatchStyledComponentsSnapshot')
+const toHaveStyle = require('./matchers/toHaveStyle')
 const styleSheetSerializer = require('./serializers/styleSheetSerializer')
 
 expect.addSnapshotSerializer(styleSheetSerializer)
 expect.extend({ toMatchStyledComponentsSnapshot })
+expect.extend({ toHaveStyle })
```

Comment:
What about `toHaveStyleRule.js` and `toHaveStyleRule.native.js` @leocreatini @Kureev?